Options.py

```diff
@@ -875,9 +875,17 @@ class ProgressionBalancing(SpecialRange):
     }
 
 
+class AllowCollect(DefaultOnToggle):
+    """Controls whether items are collected from the slot when a player does a !collect or not.
+    The impact for the collecting player is that the collector might not get all of their items, until
+     the player(s) that has disallowed collection actually completes or releases their location checks."""
```

Comment:
```suggestion
     the player(s) that has disallowed collection actually completes or releases their location checks.
     By turning it off it can result in games not syncing checks gotten between different players on 
     the same slot or if resetting a save file for certain games."""
```